commit 7c6141ebc4602262a3143c012dc2a6dbe618e4ef
Merge: 157aa7f 6cc6029
Author: ErikZalm <erik@vdzalm.eu>
Date:   Sun Feb 5 02:19:35 2012 -0800

    Merge pull request #54 from haxar/patch-1
    
    M109 EOL fix when TEMP_RESIDENCY_TIME is undefined; printer interface (p...

diff --git a/Marlin/Marlin.pde b/Marlin/Marlin.pde
index 824a57d..bbef817 100644
--- a/Marlin/Marlin.pde
+++ b/Marlin/Marlin.pde
@@ -864,6 +864,8 @@ FORCE_INLINE void process_commands()
               {
                  SERIAL_PROTOCOLLN( "?" );
               }
+            #else
+              SERIAL_PROTOCOLLN("");
             #endif
             codenum = millis();
           }
